2023-10-12/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/vmorsell/joel/2023-10-12/names"
+)
+
+func main() {
+
+	var name string
+
+	fmt.Println("what is your name?")
+
+	fmt.Scan(&name)
+
+	if name == names.GitKenobi || name == names.Vmorsell {
+		fmt.Printf("Welcome, %s \n",
+			name)
+	} else {
+		fmt.Println("You don't live here, leave")
+	}
+}
```

Comment:
I reverted 2023-10-12 to the original and moved this to 2023-10-12_2 in 7aadb0ddd61a73e270f685ed1fed2bb0f8ce9d80 and a22c78fc706c850b31eee69d31e8d9b266bc24f6